cmd/kubeadm/app/constants/constants.go

```diff
@@ -27,6 +27,10 @@ const (
 	// KubernetesDir is the directory kubernetes owns for storing various configuration files
 	KubernetesDir = "/etc/kubernetes"
 
+	PKISecretName               = "kubernetes-pki"
```

Comment:
I agree. This definitely did not feel safe when I was coding it :D. I think we need a discussion around this.